elections.md

```diff
@@ -65,40 +57,81 @@ representation is achieved. If it is impossible to find sufficient members to
 resign, the entire company’s representation will be removed and new special
 elections held. In the event of a question of company membership (for example
 evaluating independence of corporate subsidiaries) a majority of all
-non-involved Steering Committee members will decide. 
+non-involved Steering Committee members will decide.
 
-### Initial Election
+### Terms and Election Cycles
+
+Steering committee members are elected to serve two year terms.
+
+Election cycles are scheduled such that roughly half of the seats come up for
+re-election each year for purposes of continuity.  The exact number of seats
+alternates between 3 and 4, with the first 3-seat election taking place in
+2018.
+
+## Emeritus Term
+
+Members of the steering committee will graduate to becoming Emeritus members of
+the steering committee upon vacating their seat.  This confers honor on the
+recipient, acknowledging the significant contributions they have made to the
+project. Emeritus members have no binding vote, and no expectation of continued
```

Comment:
I made it explicit that Emeritus members aren't granted any additional powers nor expectations